.github/workflows/test.yml

```diff
@@ -15,6 +15,10 @@ jobs:
           - '7.0'
           - '7.1'
           - '7.2'
+          - '8.0'
+        exclude:
```

Comment:
Assuming this is excluded because Rails changed the minimum Ruby version?

Can we make that clearer via a comment?